src/CSSList/CSSList.php

```diff
@@ -148,7 +151,7 @@ private static function parseListItem(ParserState $parserState, CSSList $list)
                 return null;
             }
         } else {
-            return DeclarationBlock::parse($parserState, $list);
+            return DeclarationBlock::parse($parserState, $list) ?? false;
```

Comment:
We should cover the two functional changes in this class with (functional) tests.

Reply:
There is no functional change here.  The return value is still `false` if `DeclarationBlock::parse()` fails.  The code change here is because that now returns `null` on failure rather than `false`, but this method still needs to return `false`.